runtime/rococo/src/lib.rs

```diff
@@ -624,30 +1341,15 @@ impl parachains_initializer::Config for Runtime {
 	type WeightInfo = weights::runtime_parachains_initializer::WeightInfo<Runtime>;
 }
 
-impl paras_sudo_wrapper::Config for Runtime {}
-
-parameter_types! {
-	pub const PermanentSlotLeasePeriodLength: u32 = 365;
-	pub const TemporarySlotLeasePeriodLength: u32 = 3;
-	pub const MaxPermanentSlots: u32 = 25;
-	pub const MaxTemporarySlots: u32 = 20;
-	pub const MaxTemporarySlotPerLeasePeriod: u32 = 5;
-}
-
-impl assigned_slots::Config for Runtime {
+impl parachains_disputes::Config for Runtime {
 	type Event = Event;
-	type AssignSlotOrigin = EnsureRoot<AccountId>;
-	type Leaser = Slots;
-	type PermanentSlotLeasePeriodLength = PermanentSlotLeasePeriodLength;
-	type TemporarySlotLeasePeriodLength = TemporarySlotLeasePeriodLength;
-	type MaxPermanentSlots = MaxPermanentSlots;
-	type MaxTemporarySlots = MaxTemporarySlots;
-	type MaxTemporarySlotPerLeasePeriod = MaxTemporarySlotPerLeasePeriod;
+	type RewardValidators = ();
+	type PunishValidators = ();
+	type WeightInfo = weights::runtime_parachains_disputes::WeightInfo<Runtime>;
 }
 
 parameter_types! {
-	pub const ParaDeposit: Balance = 5 * DOLLARS;
-	pub const DataDepositPerByte: Balance = deposit(0, 1);
+	pub const ParaDeposit: Balance = 40 * UNITS;
 }
```

Comment:
Are we missing DataDepositPerByte?

This is what we have currently in Rococo:
https://github.com/paritytech/polkadot/blob/9804117d3e676aace31ed39569a4e6aec75956de/runtime/rococo/src/lib.rs#L649-L662